src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public int makeRandomTo11() {
+        return random.nextInt(10) + 1;
+    }
```

Comment:
Private ones (methods, classes) should be after public ones.



```suggestion
    private int getRandomLength() {
        return random.nextInt(MAX_LENGTH) + 1;
    }
```

![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/958dd4e0-fca3-46a5-bf1f-ca34d997ced7)

Reply:
thank you very much, I will pay attention to it now

Reply:
not fixed